sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,463 @@
+= sycl_ext_intel_fpga_task_sequence
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2022-2023 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+== Contributors
+
+// spell-checker: disable
+Jessica Davies, Intel +
+Joe Garvey, Intel +
+Robert Ho, Intel +
+Tommy Hoffner, Intel +
+Ajaykumar Kannan, Intel +
+Michael Kinsner, Intel +
+Abhishek Tiwari, Intel
+// spell-checker: enable
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 6 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+This extension also depends on the following other SYCL extension:
+
+* link:./sycl_ext_oneapi_properties.asciidoc[sycl_ext_oneapi_properties]
+* link:./sycl_ext_oneapi_kernel_properties.asciidoc[
+  sycl_ext_oneapi_kernel_properties]
+
```

Comment:
Can you also mention that there's an interaction, but not a dependence on, [sycl_ext_intel_fpga_kernel_interface_properties](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/sycl_ext_oneapi_kernel_properties.asciidoc)

Reply:
Updated.

Reply:
This has not yet been addressed.

Reply:
Abhishek has already added this line, you may be looking at an outdated version of the commits.